app/config/entity-definitions/person-identity.js

```diff
@@ -1,9 +1,7 @@
-
-
 export default {
-    code: {
+    perId: {
         domain: 'DO_ID',
-        isRequired: true
+        isRequired: false
```

Comment:
In the service in the first  PR #10 you use code and here you replace code by perId....